sequencer/src/bin/dev-rollup.rs

```diff
@@ -0,0 +1,216 @@
+use std::str::FromStr;
+
+use anyhow::Result;
+use clap::{Parser, Subcommand};
+use committable::Committable;
+use espresso_types::{
+    v0_3::{RollupRegistration, RollupRegistrationBody, RollupUpdate, RollupUpdatebody},
+    SeqTypes,
+};
+use hotshot::types::BLSPubKey;
+use hotshot_types::traits::{node_implementation::NodeType, signature_key::SignatureKey};
+use marketplace_solver::SolverError;
+
+use sequencer_utils::logging;
+use url::Url;
+use vbs::version::StaticVersion;
+
+#[derive(Debug, Parser)]
+struct Options {
+    #[clap(flatten)]
+    logging: logging::Config,
+
+    #[command(subcommand)]
+    command: Command,
+}
+
+#[derive(Debug, Subcommand)]
+enum Command {
+    Register(Register),
+    Update(Update),
+}
+
+// Options for registering a rollup
+#[derive(Parser, Debug)]
+struct Register {
+    #[clap(short, long, env = "ESPRESSO_MARKETPLACE_SOLVER_API_URL")]
+    pub solver_url: Url,
+
+    #[clap(short, long = "ns")]
+    pub namespace_id: u64,
+
+    #[clap(
+        long,
+        env = "ESPRESSO_MARKETPLACE_RESERVE_BUILDER_URL",
+        default_value_t = Url::from_str("http://localhost").unwrap()
+    )]
+    pub reserve_url: Url,
+
+    #[clap(long, default_value_t = 200)]
+    pub reserve_price: u64,
+
+    #[clap(long, default_value_t = true)]
```

Comment:
Can we change this default to false for now, since we aren't testing the auction yet.  I want to make sure we are testing rollups using reserve sequencers and not participating in the auction.